src/ng-xform/fields/dynamic-field.ts

```diff
@@ -7,6 +7,7 @@ export abstract class DynamicField<T = any> {
   public validators?: Array<ValidatorFn | null | undefined>;
   public readOnly?: boolean;
   public visibilityFn?: (val: any) => boolean;
+  public keepValueWhenHiding?: boolean;
```

Comment:
This doesn't need to be public. I can't think a case where the value should not be kept when a field is hidden

Reply:
LOL. I need this to implement the event recurrence because of the recurrence fields are hidden when a checkbox is unchecked.

Reply:
How this attribute could not be public if I need to access it from outside this object